### ISSUE TITLE
Make go back button work

### ISSUE THREAD

**Issue by Xmen097** [CA-elections/frontend#4]
https://stackoverflow.com/questions/30915173/react-router-go-back-a-page-how-do-you-configure-history

**Comment by lesves** [comment]
I agree, a non functioning go back button can ruin your user experience easily
